addons/hr_work_entry_holidays/tests/test_work_entry.py

```diff
@@ -78,16 +77,15 @@ def test_refuse_leave_work_entry(self):
 
     def test_time_week_leave_work_entry(self):
         # /!\ this is a week day => it exists an calendar attendance at this time
-        start = datetime(2015, 11, 2, 10, 0, 0)
-        end = datetime(2015, 11, 2, 17, 0, 0)
-        work_days_data = self.jules_emp._get_work_days_data_batch(start, end)
         leave = self.env['hr.leave'].create({
             'name': '1leave',
             'employee_id': self.richard_emp.id,
             'holiday_status_id': self.leave_type.id,
-            'date_from': start,
-            'date_to': end,
-            'number_of_days': work_days_data[self.jules_emp.id]['days'],
+            'request_date_from': date(2015, 11, 2),
+            'request_date_to': date(2015, 11, 2),
+            'request_unit_hours': True,
+            'request_hour_from': '11',
```

Comment:
Originally, the leave was set to be from 10 to 17, is there a reason it's 11 to 17 now?

Reply:
Yes, before when they were using date_from,to, the date_to represented the next day but in the morning, so not inclusive. In general, you'll see a lot that some of those tests have changed in order to work, due to various reasons (duration manually hardcoded, interaction with demo data etc).